Library/Homebrew/build.rb

```diff
@@ -103,6 +103,13 @@ def install
       end
     end
 
+    old_tmpdir = ENV["TMPDIR"]
+    old_temp = ENV["TEMP"]
+    old_tmp = ENV["TMP"]
+    ENV["TMPDIR"] = HOMEBREW_TEMP
+    ENV["TEMP"] = HOMEBREW_TEMP
+    ENV["TMP"] = HOMEBREW_TEMP
```

Comment:
These can be chained in Ruby, I think? 
```ruby
ENV["TMPDIR"] = ENV["TEMP"] = ENV["TMP"] = HOMEBREW_TEMP
```
Might make the duplication a little less unpleasant.